QMLComponents/boundcontrols/boundcontrolbase.h

```diff
@@ -48,9 +48,9 @@ class BoundControlBase : public BoundControl
 	std::string					getName()													const;
 	void						handleComputedColumn(const Json::Value& value);
 
-	static Json::Value			_createTableOption(const Terms& terms, const ListModel::RowControlsValues& componentValuesMap, const std::string& keyValue, const std::string& keyLabel, bool hasInteraction, bool keyHasVariables);
-	void						_setTableValue(const Terms& terms, const ListModel::RowControlsValues& componentValuesMap, const std::string& keyValue, const std::string& keyLabel, bool hasInteraction, bool keyHasVariables = false);
-	void						_readTableValue(const Json::Value& value, const std::string& keyValue, const std::string& keyLabel, bool hasMultipleTerms, Terms& terms, ListModel::RowControlsValues& allControlValues, const Terms& sourceTerms = Terms());
+	static Json::Value			_createArrayOption(const Terms& terms, const ListModel::RowControlsValues& componentValuesMap, const std::string& keyValue, const std::string& keyLabel, bool hasInteraction, bool keyHasVariables);
```

Comment:
looks like this function belongs on either `ListModel::RowControlsValues` or `Terms` no?